Formula/center-im.rb

```diff
@@ -1,9 +1,8 @@
 class CenterIm < Formula
   desc "Text-mode multi-protocol instant messaging client"
   homepage "https://github.com/petrpavlu/centerim5"
-  url "https://www.centerim.org/download/releases/centerim-4.22.10.tar.gz"
-  sha256 "93ce15eb9c834a4939b5aa0846d5c6023ec2953214daf8dc26c85ceaa4413f6e"
-  revision 2
+  url "https://www.centerim.org/download/releases/centerim-5.0.1.tar.gz"
+  sha256 "8d0689facd5e0b8803fb81ffa168c3c2022723f9d9e802ab0ea78b2157665f56"
```

Comment:
This is not a real archive anymore:

> Eviction notice
This site has been shut down and the domain is going to be removed, so please redirect yourself to the CenterIM 5 github homepage for future reference.
>
> Goodbye and thanks for all that fish.

Let's try to download it from Github as suggested:
```suggestion
  url "https://github.com/petrpavlu/centerim5/releases/download/v5.0.1/centerim5-5.0.1.tar.gz"
  sha256 "b80b999e0174b81206255556cf00de6548ea29fa6f3ea9deb1f9ab59d8318313"
```

And we can remove `livecheck` block below.